.gitignore

```diff
@@ -22,3 +22,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 .idea
+yarn.lock
```

Comment:
remove this, the yarn.lock is useful in the repo, there is an great explanation about this in yarn docs:
https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/